src/functionalTest/java/io/knotx/stack/functional/KnotxFragmentsDebugDataWithHandlebarsTest.java

```diff
@@ -94,6 +97,15 @@ void requestPage(VertxTestContext testContext, Vertx vertx,
             JsonObject nodeLog = new JsonObject(result);
             assertNotNull(nodeLog);
             assertEquals(8, nodeLog.getJsonObject("log").getJsonArray("operations").size());
+            JsonArray nodes = nodeLog.getJsonObject("log").getJsonArray("operations");
+            assertEquals("fetch-user-info", nodes.getJsonObject(0).getString("node"));
+            assertEquals("fetch-payment-providers", nodes.getJsonObject(1).getString("node"));
+            assertEquals("fetch-offers", nodes.getJsonObject(2).getString("node"));
+            assertEquals("fetch-offers-fallback", nodes.getJsonObject(3).getString("node"));
+            assertEquals("fetch-delivery-options-cb", nodes.getJsonObject(4).getString("node"));
+            assertEquals("fetch-delivery-timeout", nodes.getJsonObject(5).getString("node"));
+            assertEquals("composite", nodes.getJsonObject(6).getString("node"));
```

Comment:
Composite nodes entries can be different. See the example below:
- composite `[A -> B, C -> D]`
- possible outputs: [A, B, C, D], [C, D, A, B]
Please fix the validation rules - check `io.knotx.fragments.engine.TaskEngineCompositeNodeTest#expectSuccessEventLogEntry`.

Reply:
Unfortunately, there's no possibility to get a more detailed message regarding this entry